platform/admin/core/admin-core-logviewer/pom.xml

```diff
@@ -79,7 +79,7 @@
                 <configuration>
                     <instructions>
                         <Web-ContextPath>/admin/logviewer</Web-ContextPath>
-                        <_wab>${project.build.directory}/webapp/</_wab>
```

Comment:
@harrison-tarr did you comment these out because they fail the build and they need to be put back at some point?